diff --git a/cdisc_rules_engine/enums/standard_types.py b/cdisc_rules_engine/enums/standard_types.py
index fd4b49b..4571eb3 100644
--- a/cdisc_rules_engine/enums/standard_types.py
+++ b/cdisc_rules_engine/enums/standard_types.py
@@ -10,5 +10,11 @@ class StandardTypes(BaseEnum):
     SENDIG_DART = "sendig-dart"
     SENDIG_GENETOX = "sendig-genetox"
     ADAMIG = "adamig"
+    ADAM_ADAE = "adam-adae"
+    ADAM_MD = "adam-md"
+    ADAM_NCA = "adam-nca"
+    ADAM_OCCDS = "adam-occds"
+    ADAM_TTE = "adam-tte"
+    ADAM_POPPK = "adam-poppk"
     TIG = "tig"
     USDM = "usdm"
